tests/e2e/framework/framework.go

```diff
@@ -182,7 +182,7 @@ func (c *CommonConfig) saveLogs(r int) error {
 		glog.Errorf("Could not create status file. Error %s", err)
 		return err
 	}
-	if r != 0 && *logProvider == "stackdriver" { // fetch logs only if tests failed
+	if *logProvider == "stackdriver" { // fetch logs only if tests failed
```

Comment:
Please eemove logProvider here as well as calls from the shell script.

Reply:
Done